### DISCUSSION TITLE
Use Mailchannels via Cloudflare for Free Email

### DISCUSSION THREAD

**Discussion by mukhtharcm** [pocketbase/pocketbase#4296] [Q&A] [answered]
Mailchannels and Cloudflare workers have a partnership that enables us to send email without any cost if you're using Mailchannels API via a Cloudflare Worker.
1. https://community.cloudflare.com/t/send-email-from-workers-using-mailchannels-for-free/361973/18
2. https://blog.cloudflare.com/sending-email-from-workers-with-mailchannels#:~:text=it%20is%20completely-,free,-to%20send%20email
3. https://blog.mailchannels.com/mailchannels-enables-free-email-sending-for-cloudflare-workers-customers/
4. https://github.com/Sh4yy/cloudflare-email

Is there any way to use this instead of the SMTP services in pocketbase?

**Comment by ganigeorgiev** [comment, marked as answer]
> Is there any way to use this instead of the SMTP services in pocketbase?

No, at least not out of the box.

If you are looking for free email options you could check the free plans of [Mailjet](https://www.mailjet.com/pricing/), [Brevo](https://www.brevo.com/pricing/), [Sendgrid](https://sendgrid.com/en-us/pricing), [Mailtrap Sending](https://mailtrap.io/pricing/), and probably others.

**Reply by mukhtharcm** [reply]
Thank you!

**Comment by Spikeysanju** [comment]
@mukhtharcm FYI, Cloudflare discontinued MailChannels APIs by June 30th, 2024. [Official source](https://support.mailchannels.com/hc/en-us/articles/26814255454093-End-of-Life-Notice-Cloudflare-Workers#:~:text=Unfortunately%2C%20we%20regret%20to%20announce,an%20alternative%20email%20delivery%20solution.)
